src/webcomponents/clinical/clinical-analysis-update.js

```diff
@@ -338,18 +373,21 @@ class ClinicalAnalysisUpdate extends LitElement {
                                     let panelHtml = "-";
                                     if (panels?.length > 0) {
                                         panelHtml = html`
-                                            ${panels.map(panel => {
-                                                if (panel.source?.project?.toUpperCase() === "PANELAPP") {
-                                                    return html`
-                                                        <div style="margin: 5px 0px">
-                                                            <a href="${BioinfoUtils.getPanelAppLink(panel.source.id)}" target="_blank">
-                                                                ${panel.name} (${panel.source.project} v${panel.source.version})
-                                                            </a>
-                                                        </div>`;
-                                                } else {
-                                                    return html`<div>${panel.id}</div>`;
-                                                }
-                                            })}`;
+                                            ${
+                                                panels.map(panel => {
+                                                    if (panel.source?.project?.toUpperCase() === "PANELAPP") {
+                                                        return html`
+                                                            <div style="margin: 5px 0">
+                                                                <a href="${BioinfoUtils.getPanelAppLink(panel.source.id)}" target="_blank">
+                                                                    ${panel.name} (${panel.source.project} v${panel.source.version})
+                                                                </a>
+                                                            </div>`;
+                                                    } else {
+                                                        return html`<div>${panel.id}</div>`;
+                                                    }
+                                                })
+                                            }
+                                        `;
```

Comment:
Please move `panels.map(panel => {` to the previous line and fix the indentation:
```
${panels.map(panel => {
    ....
})}
```

Reply:
After several attempts, this is the only approach that I have found to avoid the ESLint error shown below. I am not happy to prevent ESLint from blocking the commit  by using the hook --no-verify. It might be possible to configure ESLint in a different way. I will follow your advise @jmjuanes , what is your preference for now?

![Screenshot 2022-10-26 at 10 50 08](https://user-images.githubusercontent.com/109655106/197980579-b5b187f7-eaca-4da5-ab20-6a3aa6f888a8.png)

Reply:
You can safetly remove the `html` wrapper for the `panels.map`, as is wrapped at the end of the function inside a `div` tag:

```js
render: panels => {
    let panelHtml = "-";
    if (panels?.length > 0) {
        panelHtml = panels.map(panel => {
            ...
        });
    }
    return html`<div>${panelHtml}</div>`;
},
```